airflow/models/dagbag.py

```diff
@@ -419,8 +419,6 @@ def collect_dags(
                 dag_id_names = str(dag_ids)
 
                 td = timezone.utcnow() - ts
-                td = td.total_seconds() + (
-                    float(td.microseconds) / 1000000)
```

Comment:
For this specific file/change, in order to avoid breaking other logics in other files, I would like to suggest to change in `Stats.timing()` line instead.

I.e., revert your change in line 422, then change

```python
filename = file_stat.file.split('/')[-1].replace('.py', '')
Stats.timing('dag.loading-duration.{}'.
             format(filename),
             format(filename),
             file_stat.duration)
```

into

```python
filename = file_stat.file.split('/')[-1].replace('.py', '')
Stats.timing('dag.loading-duration.{}'.
             format(filename),
             format(filename),
             timedelta(seconds=file_stat.duration))
```
(`timedelta` needs to be imported)


Let me know your opinion? Cheers